share/commands/remote-run

```diff
@@ -20,17 +20,18 @@
 
 VERBOSITY=""
 COLOR="-C"
+DEBUG_CLASS="-D debug"
 
 while getopts "iIvdc" opt; do
   case $opt in
     i|I)
-      VERBOSITY="-I -D debug"
+      VERBOSITY="-I ${DEBUG_CLASS}"
       ;;
     v)
-      VERBOSITY="-v"
+      VERBOSITY="-v ${DEBUG_CLASS}"
       ;;
     d)
-      VERBOSITY="-d"
+      VERBOSITY="-d ${DEBUG_CLASS}"
       ;;
     c)
       COLOR=""
```

Comment:
We could also modify the agent-run wrapper to be able to parse remote-run output. I don't know if it is worth it, what do you think?